### PR TITLE
chore: add docker-compose for nginx

### PR DESCRIPTION
**Pull request by ytvnr**

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-nstmloybgl.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/docker-compose-nginx/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

File: docker/quick-setup/nginx/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# MongoDB
```
Comment (marcambier) [inline docker/quick-setup/nginx/README.md]:
```suggestion
# Nginx
```

Comment (ytvnr) [reply docker/quick-setup/nginx/README.md]: 🙈

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-ddeedujyec.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/docker-compose-nginx/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Review by marcambier — APPROVED** [review]

**Pull request merged**